bridge/client/environments/environment.ts

```diff
@@ -1,3 +1,16 @@
-export const environment = {
-  production: false
-};
+import {DynamicEnvironment} from "./environment.dynamic";
+
+class Environment extends DynamicEnvironment {
+
+  public appTitle: string;
+  public logoUrl: string;
+  public logoInvertedUrl: string;
```

Comment:
Remove unused variables `appTitle`, `logoUrl` and `logoInvertedUrl`. They are accessed via `this.config.`